src/pcareconstruction.jl

```diff
@@ -0,0 +1,83 @@
+export PCAEmbedding
+
+"""
+	 PCAEmbedding <: AbstractSpatialEmbedding
+	 PCAEmbedding(s, em::SpatioTemporalEmbedding) → `embedding`
+A spatio temporal delay coordinates structure with
+Principal Component Analysis as a means of dimension reduction to be used as a functor.
+
+	embedding(rvec, s, t, α)
+Operates inplace on `rvec` and reconstructs vector from spatial time series `s` at
+timestep `t` and cartesian index `α`.
+
+Takes a [`SpatioTemporalEmbedding`](@ref) and computes PCA on the time series `s`.
+
+## Keyword Arguments
+ * `pratio = 0.99` : Ratio of variances that needs to be preserved in low-dim reconstruction
+ * `maxoutdim = 25`: Upper limit for output dimension. May break `pratio` criterion
+ * `every = 1` : Speed up computation by only using every nth point. (linear indexing in space and time)
```

Comment:
I think allowing different `every` for space and time will be helpful for us for the paper as well. How hard is it?

Reply:
Probably not too difficult.
This would require writing a more sophisticated loop.

Reply:
`IterTools.takenth(xs, n)`
should solve this nicely

Reply:
Yeap, very nice. I'll think about it when I do the doc review